### PR TITLE
test(enterprise): make sure that quota check is performed by enabling quota for the various orgs

### PR DESCRIPTION
**Pull request by machi1990**
## Description

Make sure that quota check is performed by enabling quota for the various orgs

Follows up on https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1536. This also enables quota for various orgs that uses kas-installer env using the quota-list-management mechanism.

## Verification Steps

Integration tests should still pass

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA have been completed
- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
- [ ] Documentation added for the feature
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- [ ] All PR comments are resolved either by addressing them or creating follow up tasks
- [ ] Required metrics/dashboards/alerts have been added (or PR created).
- [ ] Required Standard Operating Procedure (SOP) is added.
- [ ] JIRA has been created for changes required on the client side

### CODE REVIEW

File: internal/kafka/test/integration/kafka_create_test.go [hunk 1]
```diff
@@ -614,7 +614,7 @@ func TestKafkaCreate_EnterpriseKafkas(t *testing.T) {
 	defer ocmServer.Close()
 
 	h, client, teardown := test.NewKafkaHelperWithHooks(t, ocmServer, func(acl *quota_management.QuotaManagementListConfig, c *config.DataplaneClusterConfig) {
-		acl.EnableInstanceLimitControl = false
```
Comment (machi1990) [inline internal/kafka/test/integration/kafka_create_test.go:617]: this was causing the various quota checks to be skipped.

For this test, we always want the quota checks to be performed.

Comment (ziccardi) [reply internal/kafka/test/integration/kafka_create_test.go:617]: Good catch!

**Comment by ziccardi** [comment]
Code looks good to me. Waiting for the tests to finish

**Review by ziccardi — APPROVED** [review]
Thanks!

**Pull request merged**